backend/app/requests/caching.py

```diff
@@ -38,7 +38,7 @@ def _cache(func, hint_fields, **kwargs):
         hint_values=hint_values,
     )
     # If caching is disabled or a hint is missing, call the function normally.
-    if not config.get('cache.enabled') or not all(hint_values):
+    if not config.get('cache.enabled') or any(val is None for val in hint_values):
```

Comment:
I'm still a bit confused when hint is '' and when it's None and what the two different cases mean.